### DISCUSSION TITLE
Always get KeyError when passing a configurable param via curl or RemoteRunnable but works fine invoking the chain directly.

### DISCUSSION THREAD

**Discussion by ildiopantofola** [langchain-ai/langserve#508] [Q&A]
I have a langchain LLM chain linked to a route of a FastAPI server. I used langserve add_route() method. As I have session_id passed as "configurable" param and everything works when I invoke the chain directly like: 

> chain.invoke({"question":user_input},config={"configurable": {"session_id": SESSIONID}})

But if I want to use curl or calling using RemoteRunnable, I always get a **KeyError: 'session_id'**. 
I tried many ways but with no success:

>chat = RemoteRunnable("http://0.0.0.0:8000/chainroute") 
> chat.invoke({"question":"......"},config={"configurable": {"session_id": 1234567890}})

> curl --no-buffer -X 'POST' 'http://0.0.0.0:8000/chainroute/invoke' -H 'accept: text/plain' -H 'Content-Type: application/json' -d '{"input":{"question": "....", "session_id": "1234567890" }}'

> curl --no-buffer -X 'POST' 'http://0.0.0.0:8000/chainroute/invoke' -H 'accept: text/plain' -H 'Content-Type: application/json' -d '{"input":{"question": "....",{"configurable":{"session_id":"1234567890"}}}}'

> curl --no-buffer -X 'POST' 'http://0.0.0.0:8000/chainroute/stream' -H 'accept: text/plain' -H 'Content-Type: application/json' -d '{"input":{"question":"...."},"config":{"configurable":{"session_id":"1234567890"}}}'

But none is working, it never recognizes the session_id param. What am I missing?

**Comment by eyurtsev** [comment]
Looks correct based as I can tell, so would need more context to help.

I'd recommend testing with this example: https://github.com/langchain-ai/langserve/tree/main/examples/chat_with_persistence

It's working for me and passes session ID in exactly the same way. Perhaps it'll help identify the issue -- I'm guessing there's some minor typo somewhere? (Try maybe adding a trailing `slash` to chainroute to see if it helps -- I don't think it should and also is probably the wrong thing to do -- but worth checking quickly)
